package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "web1",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "eslint .",
+    "format": "prettier --write .",
```

Comment:
there should be 2 ways of linters/formaters invocation: intrusive and not. Probably we missing something here